src/main/webapp/profile.js

```diff
@@ -40,6 +42,12 @@ function submitProfileForm(methodType, redirectUrl) {
     });
 }
 
+function validateProfileForm() {
```

Comment:
Where is this called?

Reply:
![image](https://user-images.githubusercontent.com/34817835/88331445-fa36b100-ccfa-11ea-846e-409d0c3b9052.png)
When they submit on profile or registration page